InspectUITests/shift page.swift

```diff
@@ -35,15 +35,19 @@ class shift_page: XCTestCase {
     
     func login() {
         let app = XCUIApplication()
-        let loginButton = app.buttons["Login with IDIR"]
+//        let loginButton = app.buttons["Login with IDIR"]
```

Comment:
I think you were going to undo this?

Reply:
oh this is to get the testing working. but now that i think about it, we can get away with environment variables so we can get away with that, ill disable the bceid logins